ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl.hs

```diff
@@ -88,6 +88,7 @@ withDB ::
      , HasHardForkHistory blk
      , ConvertRawHash blk
      , SerialiseDiskConstraints blk
+     , LedgerDB.LedgerSupportsV2LedgerDB blk
```

Comment:
This qualified imported name contains `LedgerDB` twice. Maybe the prefix isn't necessary?